internal/resolvers/default/thumbnail.go

```diff
@@ -80,7 +80,7 @@ func doThumbnailRequest(urlString string, r *http.Request) (interface{}, time.Du
 		if err != nil {
 			return nil, cache.NoSpecialDur, err
 		}
-		if contentLengthBytes > resolver.MaxContentLength {
+		if contentLengthBytes > GetMaxContentLength() {
```

Comment:
Also needs to be multiplied (at this point it probably does make sense to multiply it inside `GetMaxContentLength()` and maybe rename it to make more sense like `GetMaxContentLengthBytes()`